src/graphql/root/mutation/intraledger-payment-send.ts

```diff
@@ -44,7 +44,9 @@ const IntraLedgerPaymentSendMutation = GT.Field({
     }
 
     const btcWalletValidated = await validateIsBtcWalletForMutation(walletId)
-    if (btcWalletValidated != true) return btcWalletValidated
+    if (btcWalletValidated instanceof Error) {
```

Comment:
shoudn't this check be in `app/`?

Reply:
The validation is tied to the specific query/mutation it's being called in. Not sure if app layer would have this context to apply the check.

![image](https://user-images.githubusercontent.com/17693119/210078997-244e586d-44ee-4b75-b90a-cebb2808b8fd.png)

Reply:
Actually, I guess we could differentiate this by making new app layer use-case methods. I can do that here.

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/2105/commits/02fe1376f8d159b9ade51184449d025dd1e7c8ba